distributed/scheduler.py

```diff
@@ -282,30 +283,54 @@ class WorkerState:
 
     # XXX need a state field to signal active/removed?
 
+    _actors: set
+    _address: str
```

Comment:
It seems like `address` should be a `str`, but ran into a few issues in the tests with that. Submitted PR ( https://github.com/dask/distributed/pull/4296 ) to hopefully address those.

Reply:
It sounds like `int` is allowable in some cases. Though we can work to constrain this to a `str`. For now maybe we can relax this to `object` while we figure that out and then tighten this down later.

```suggestion
    _address: object
```